### DISCUSSION TITLE
[Question] Preview of PDF

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2385] [Q&A] [answered]
### Question

Good morning. I want to preview pdf files with flet, files that I am generating through html and jinja2. The idea is to be able to see the information before saving the pdf. Is there any way to do it? thank you

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
There's no way to do that atm.

**Comment by fsmosca** [comment]
Why not convert it to markdown, and use flet's markdown control to preview it?
